app/views/layouts/_test_results.html.erb

```diff
@@ -28,5 +28,6 @@
   </ul>
 <% end %>
 
-
-
+<% unless contextualization.result.blank? %>
```

Comment:
Maybe it would be a better idea to replace this method: https://github.com/mumuki/mumuki-laboratory/blob/master/app/helpers/contextualization_result_helper.rb#L24 for a template which handles this scenario in a better way, and keep both _results and _tests_results templates cleaner.
What do you think?

Reply:
I think it's a great idea. I did it in https://github.com/mumuki/mumuki-laboratory/pull/1238/commits/0d6f14db6f305fe8f563ba73865f57af1b11ca5a